modin/pandas/test/test_io.py

```diff
@@ -456,6 +457,104 @@ def test_from_json_lines():
     teardown_json_file()
 
 
+def test_from_json_string():
+    json_string = """[{"project": "modin"}]"""
+    with pytest.warns(UserWarning):
+        modin_df = pd.read_json(json_string)
+    df_equals(modin_df, pandas.read_json(json_string))
```

Comment:
In the case we can remove `test_from_json_bytesio`, 
```suggestion
@pytest.mark.parametrize("data", [
    """[{"project": "modin"}]""",
    BytesIO(b"""[{"project": "modin"}]"""),
])
def test_read_json(data):
    with pytest.warns(UserWarning):
        modin_df = pd.read_json(data)
    df_equals(modin_df, pandas.read_json(data))
```